.travis.yml

```diff
@@ -5,6 +5,7 @@ env:
     - APP_SECRET_TOKEN=b2724973fd81c2f4ac0f92ac48eb3f0152c4a11824c122bcf783419a4c51d8b9bba81c8ba6a66c7de599677c7f486242cf819775c433908e77c739c5c8ae118d
   matrix:
     - DATABASE_ADAPTER=mysql2
+    - DATABASE_ADAPTER=mysql2 FARADAY_HTTP_BACKEND=net_http
```

Comment:
Not sure, but at least I was able to find another issue by this build option and I'm working on it.